needles/desktop_terminal.json

```diff
@@ -10,6 +10,14 @@
       "width": 1024,
       "height": 728,
       "type": "match"
+    },
+    {
+      "": "Terminal exlusion zone due to desktop shift bug",
+      "xpos": 144,
+      "ypos": 82,
+      "width": 735,
+      "height": 481,
+      "type": "match"
```

Comment:
The comment says this is an exclusion, but the type is `match` not `exclude`. What’s this needed for? If there’s a bug in the desktop, it needs to be fixed, not written into the tests. (Or, if there is a good justification for temporarily working around a bug in the tests, the comment needs to point to the relevant Phabricator issue for resolving the bug properly.)

Reply:
(Dropped)